qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1221,13 +1221,13 @@ def quantum_successors(self, node):
                 successors.append(successor)
         return successors
 
-    def _remove_op_node(self, node):
+    def remove_op_node(self, node):
```

Comment:
Should we check that `node` is of type `op`?

Reply:
True! Fixed in dc7631f8